### PR TITLE
Feature/studio

### PR DESCRIPTION
**Pull request by pranay2401**
# CHANGELOG
* Added commands to build studio module
* Moved common content from im-ott to im-components
* Added Landing page with basic skeleton and appLoader

**Note**: Please review and merge after #18  so that common changes are overlapped.

### CODE REVIEW

File: packages/im-components/src/molecules/logo/Logo.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <Link to={"/"} className={styles.logo} >
+        <VideoCameraFilled className={styles.image} style={{ fontSize:"250%" }} />
+        <div className={styles.text} >{text}</div>
+      </Link>
+    </>
+  );
+}
+
+Logo.propTypes = {
+  text: PropTypes.string
```
Comment (ukirderohit) [inline packages/im-components/src/molecules/logo/Logo.js:27]: children should be props

File: packages/im-components/src/molecules/logo/logo.module.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+.logo {
+  display: inline-flex !important;
+  color: #000;
+  align-items: center;
+}
+
+.logo:hover {
+  color: #000
+}
+
```
Comment (ukirderohit) [inline packages/im-components/src/molecules/logo/logo.module.scss:16]: same comments like before

**Review by ukirderohit — CHANGES_REQUESTED** [review]

File: packages/im-components/src/molecules/logo/Logo.js (hunk 1 above)
Comment (pranay2401) [reply packages/im-components/src/molecules/logo/Logo.js:27]: Will pull changes from develop once #18 is merged. updated it there.

File: packages/im-components/src/molecules/logo/logo.module.scss (hunk 2 above)
Comment (pranay2401) [reply packages/im-components/src/molecules/logo/logo.module.scss:16]: Will pull changes from develop once #18  is merged. updated it there.

**Pull request merged**